lib/phoenix_test/playwright/connection.ex

```diff
@@ -161,7 +168,8 @@ defmodule PhoenixTest.Playwright.Connection do
   end
 
   defp log_js_error(state, %{method: :page_error} = msg) do
-    Logger.error("Javascript error: #{inspect(msg.params.error)}")
+    location = get_in(msg, [:params, :location, Access.key(:url, "unknown location")])
```

Comment:
I wonder if we should just omit the location info instead of appending `unknown location`.
What does the browser-internal console show in this case?

Reply:
Yes, you're right. I'm not sure there's a way to trigger a empty/nil location at all, because there will always be a loadable location. An empty href does not result in a request

<link rel="stylesheet" href="" type="text/css"> -> No error in Safari, Chrome, Firefox

I've removed `unknown location` and added the line number conditionally if it's included in the location